va/va_test.go

```diff
@@ -641,13 +648,16 @@ func TestMultiVALogging(t *testing.T) {
 		if len(rva1Log.GetAllMatching(`"Perspective":"dev-arin"`)) >= 1 &&
 			len(rva1Log.GetAllMatching(`"RIR":"ARIN"`)) >= 1 &&
 			len(rva2Log.GetAllMatching(`"Perspective":"dev-ripe"`)) >= 1 &&
-			len(rva2Log.GetAllMatching(`"RIR":"RIPE"`)) >= 1 {
+			len(rva2Log.GetAllMatching(`"RIR":"RIPE"`)) >= 1 &&
```

Comment:
FWIW I would just remove the Perspective and RIR from the audit log event, and this test, altogether. We never collect audit log events from the the remoteVAs and logging the perspective in-DC alone doesn't satisfy the MPIC requirements.